app/code/Magebit/Faq/Model/Data/Question.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @copyright Copyright (c) 2024 Magebit, Ltd. (https://magebit.com/)
+ * @author    Magebit <info@magebit.com>
+ * @license   MIT
+ */
+
+declare(strict_types=1);
+
+namespace Magebit\Faq\Model\Data;
+
+use Magebit\Faq\Api\Data\QuestionInterface;
+use Magento\Framework\DataObject;
+
+class Question extends DataObject implements QuestionInterface
+{
+    /**
+     * Getter for Id.
+     *
+     * @return int|null
+     */
+    public function getId(): ?int
+    {
+        return $this->getData(self::ID) === null ? null
```

Comment:
no need to check if null.

just return $this->getData(self::ID)

if the id is set, it will return it and if it is null, it would return null just as in your code